packages/fast-usdc-contract/src/utils/cctp.ts

```diff
@@ -0,0 +1,14 @@
+import type { AccountId, ChainHub } from '@agoric/orchestration';
+import { parseAccountId } from '@agoric/orchestration/src/utils/address.js';
+
+export const makeIsCctpSupported =
```

Comment:
I like the composition. I'm not sure in this case it's better than method on general utility object. Maybe not ChainHub since it's about chains instead of accounts. We could consider an AccountUtils object that takes a ChainHub and has various wrappers like this.

Thinking out loud. This is fine for now.